gnomad/utils/annotations.py

```diff
@@ -1364,11 +1383,21 @@ def _sum_or_diff_fields(
                     ann: (
                         hl.case()
                         .when(set_negatives_to_zero, hl.max(x[ann], 0))
-                        .or_error(negative_value_error_msg)
+                        .or_error((negative_value_error_msg, "freq"))
```

Comment:
Just want to confirm this works and doesn't need to be `negative_value_error_msg % "freq"` instead